appveyor.yml

```diff
@@ -1,4 +1,4 @@
-os: Windows Server 2012
+image: Visual Studio 2019
```

Comment:
Why is OS removed?

Reply:
The os is directly bound to the image. The first attempt was to use
```
os: Windows Server 2019
```
but it fails with `Build worker image not found: Windows Server 2019`
https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/31741789/job/j0f3kcr1rjv4v7oq/messages

So i removed the `os` label and add `image`.

Reply:
The `os` tag is absent in the  [Reference](https://www.appveyor.com/docs/appveyor-yml/). I suppose it is deprecated now.

Reply:
os name is the same
https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/31947839/job/dic8725umb3lrsic#L22

```
mvn --version
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T17:29:23+00:00)
Maven home: C:\maven\apache-maven-3.2.5
Java version: 1.8.0_221, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows server 2019", version: "10.0", arch: "amd64", family: "dos"
java -version
```